api_cgn_operator_search.yaml

```diff
@@ -0,0 +1,113 @@
+swagger: "2.0"
+info:
+  version: 1.0.0
+  title: Carta Giovani Nazionale Operator Search API
+host: localhost
+basePath: /api/v1
+schemes:
+  - https
+security:
+  - Bearer: []
+paths:
+  "/cgn-operator-search/online-merchants":
+    post:
+      operationId: getOnlineMerchants
+      summary: List of online merchants
+      description: |
+        List of online merchants alphabetically ordered
+      parameters:
+        - in: body
+          name: body
+          required: false
+          schema:
+            $ref: '#/definitions/OnlineMerchantSearchRequest'
+      responses:
+        "200":
+          description: List of online merchants for the given query parameters
+          schema:
+            $ref: "#/definitions/OnlineMerchants"
+        "500":
+          description: Service unavailable.
+          schema:
+            $ref: "#/definitions/ProblemJson" 
+
+  "/cgn-operator-search/offline-merchants":
+    post:
+      operationId: getOfflineMerchants
+      summary: List of merchants with a physical address
+      description: |
+        List of merchants with physical address, ordered by distance from the user by default
+      parameters:
+        - in: body
+          name: body
+          required: true
+          schema:
+            $ref: '#/definitions/OfflineMerchantSearchRequest'
+      responses:
+        "200":
+          description: List of physical merchants for the given query parameters
+          schema:
+            $ref: "#/definitions/OfflineMerchants"
+        "500":
+          description: Service unavailable.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+
+  "/cgn-operator-search/merchants/{merchantId}":
+    parameters:
+      - name: merchantId
+        in: path
+        type: string
+        minLength: 1
+        required: true
+        description: Merchant Identifier
+    get:
+      operationId: getMerchant
+      summary: Retrieve merchant details
+      description: |
+        Retrieve merchant details together with published discounts
+      responses:
+        "200":
+          description: Found
+          schema:
+            $ref: "#/definitions/Merchant"
+        "404":
+          description: Operator Not Found
+        "500":
+          description: Service unavailable.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+definitions:
+  ProblemJson:
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v17.3.0/openapi/definitions.yaml#/ProblemJson"
```

Comment:
```suggestion
    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v21.0.1/openapi/definitions.yaml#/ProblemJson"
```

Reply:
done in https://github.com/pagopa/io-backend/pull/804/commits/3deb2fdb462b04d5e9ad2a5aa310600ce6cffe08 thx